docs/docs/quickstart/nebari-101.md

```diff
@@ -0,0 +1,19 @@
+---
+id: nebari-101
+---
+
+# Nebari 101
+
+## What Nebari is
+
+## What Nebari isn't
+
+## Main features
+
+## Nebari components
+
+## Nebari OnPrem and Nebari
+
+## Use cases
+
+## Why use Nebari?
```

Comment:
I think I would put this section right below `What Nebari isn't` -- this is a "quick blast to help me orient on Nebari" section, whereas the other sections from `Main features` to `Use cases` are more heavy-duty info.

Reply:
This is out of the scope of this issue, but I agree.